src/client/Microsoft.Identity.Client/Platforms/Features/DesktopOS/WindowsNativeUtils.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Identity.Client.Platforms.Features.DesktopOs
```

Comment:
@bgavrilMS  Hmm should I put these into a different namespace? Do we have one for public helpers?